ivadomed/loader/loader_files.py

```diff
@@ -0,0 +1,73 @@
+import copy
```

Comment:
Correct. I will remove this file. I think the original plan was to have a dedicated loader per each Dataset type (files vs Bids vs Regex) but that has since been superseded by the JSON specific FIlesDataset and future BIDsDataset/RegexDataset design.